libs/email-builder/src/lib/components/RenderTagAndSignUpPageCreationMessage.tsx

```diff
@@ -54,24 +65,38 @@ export const RequestTagCreationMessage = ({ pageLink, newsletter }: Props) => {
 				</div>
 			)}
 
+			<h2> Create sign-up page </h2>
+			<NewsletterPropertyTable
+				newsletter={newsletter}
+				properties={['name', 'signUpHeadline', 'signUpDescription']}
+			/>
+			{embargoDate && (
+				<p>
+					Please add an embargo so that the sign up page should not go live
+					until {embargoDate}.
+				</p>
+			)}
+			{!embargoDate && <p>The page can go live immediately.</p>}
```

Comment:
This is more of a product question, but would w want the page to be launched before the newsletter is set to live? That woudl mean the sign-up page would have no form rendered

Reply:
Will clarify with Sophie - thank you